### ISSUE TITLE
Error opening files after Update >5.1.0.115

### ISSUE THREAD

**Issue by derBobby** [ONLYOFFICE/docker-onlyoffice-owncloud#19]
Hi folks,

after updating to the newest ONLYOFFICE docker image available I can't open files in nextcloud anymore. When opening a file this message pops up after several ten seconds.
![oo](https://user-images.githubusercontent.com/11094004/39951981-9a3197be-558f-11e8-987c-b5bf8e56458b.JPG)

My script to start the container (not using compose obviously) is this, where I just edited the version number. The default.json is unchanged.
```
docker create -t --rm --name nextcloud-oo \
        -v <path>/default.json:/etc/onlyoffice/documentserver/default.json \
        --network nextcloud \
        onlyoffice/documentserver:5.1.3.35
```

The logs in /var/log/onlyoffice/documentserver/ show this error message.
```
Error: Configuration property "editor" is not defined
    at Object.Config.get (/var/www/onlyoffice/documentserver/server/DocService/node_modules/config/lib/config.js:202:11)
    at Object.<anonymous> (/var/www/onlyoffice/documentserver/server/DocService/sources/DocsCoServer.js:40:50)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/var/www/onlyoffice/documentserver/server/DocService/sources/gc.js:17:20)
/var/www/onlyoffice/documentserver/server/DocService/node_modules/config/lib/config.js:202
    throw new Error('Configuration property "' + property + '" is not defined');
    ^
```

Nextcloud v13.0.0
Nextcloud ONLYOFFICE app v1.3.0
Last working ONLYOFFICE docker image 5.1.0.115
Non-working ONLYOFFICE docker images 5.1.1.23 5.1.2.9 5.1.3.35

Need any further details?

**Comment by derBobby** [comment]
Well great. After submitting the issue I went further analysing what happens with the default.json on updates and found a solution myself.

Seems like it is not the best idea to mount the file as volume. Even with file permissions 777 the container did not update the content of the file as needed by the newer version.

So what I did was a manual diff and edit of:
- my modified file containing my secret and
- the new not mounted file from inside the running container of the new image version. The error '_Configuration property "editor" is not defined_' fits to the result of the diff:
```
55a56
>         "sockjsUrl": "",
58a60
>         "editor_settings_spellchecker_url": "/spellchecker",
190,200d191
<       },
<       "editor": {
<         "spellcheckerUrl": "/spellchecker",
<         "reconnection": {
<           "attempts": 50,
<           "delay": "2s"
<         }
<       },
<       "sockjs": {
<         "sockjs_url": "",
<         "websocket": true
```

Remaining question for me: Should I not mount the default.json as volume? Should I instead run a 'sed snippet' for injecting my secrets after starting the container?
